internal/virtualmtime/virtualmtime.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2014 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package virtualmtime
+
+import (
+	"time"
+)
+
+type VirtualMtimeRepo interface {
```

Comment:
It's a bit unusual compared to how the rest of the code is set up, and would normally be declared on the using side when necessary... Unless there are multiple implementations in the exporting package.